interface-definitions/include/dhcp/ddns-dns-server.xml.i

```diff
@@ -0,0 +1,19 @@
+<!-- include start from dhcp/ddns-dns-server.xml.i -->
+<tagNode name="dns-server">
```

Comment:
In Kea's config, the `ip-address` is the required option and the "primary key" of server entries. It also has `hostname` key which is currently ignored, but that we could add to make servers easier to identify visually.

My feeling is that the server number here is just extra information — as far as I can see from [the docs](https://kea.readthedocs.io/en/kea-2.2.0/arm/ddns.html#adding-reverse-dns-servers), the order of servers in the list has no special meaning to Kea.

Unless I'm missing anything here, I would make it `set dns-server ns1.example.com ip-address 192.0.2.10` (which would generate `{'ip-address': '192.0.2.10', 'hostname': 'ns1.example.com'}`).